pkg/collector/corechecks/containers/docker/docker.go

```diff
@@ -51,7 +51,7 @@ func updateContainerRunningCount(images map[string]*containerPerImage, c *contai
 	var containerTags []string
 	var err error
 
-	if c.Excluded {
+	if c.Excluded || c.State != containers.ContainerRunningState {
```

Comment:
could you add the same comment than in your PR description to explain why terminated container are considered as "excluded".

I also wondering if "pending" will go also on this case?

Reply:
Yeah, pending will get into this branch, but they'll be ignored later as we only count containers that are either stopped or running. It would be a minor optimization to ignore them completely, I can add it if you think it's relevant. Either way, I added the comment as you requested 👍